spec/views/fields/edit_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'fields/edit.haml' do
+  before :each do
+    @form_draft = create :form_draft
+    @field = create :field, form_draft: @form_draft
+  end
+  it 'contains the name of the field' do
+    render
+    expect(rendered).to have_tag 'h1', with: { class: 'title' }
```

Comment:
This doesn't test for the name of the field, just the presence of the h1 tag which should contain it